_about/notes/0.5.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use Kubernetes 1.9's mutating webhook functionality to provide automatic
+pod-level proxy injection. Automatic injection requires Kubernetes 1.9 or beyond and
+therefore doesn't work on older versions. [Learn more]({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection)
+
+- **Alpha2 Traffic Rules**. We've started to add support for a new model for routing and destination rules. You can see the updated
```

Comment:
Revised Traffic Rules: Based on user feedback, we have made significant changes to Istio's traffic management (routing rules, destination rules, etc.). We would love your early feedback while we polish this in the coming weeks....